### PR TITLE
1744 by Inlead: Moving ting_search_carousel into panels.

### PR DESCRIPTION
**Pull request by bozeak**
http://platform.dandigbib.org/issues/1744

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 7b2f5cb890355fac5dd5563c48f5dcb547fa270e
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 4d5cc06d72185f461b2e4a0544439e1c919d75d7
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 01df08ed628122d74877eaf2f366f21199f1fede
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 7b2f5cb890355fac5dd5563c48f5dcb547fa270e
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 4d5cc06d72185f461b2e4a0544439e1c919d75d7
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 01df08ed628122d74877eaf2f366f21199f1fede
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ting_search_carousel/ting_search_carousel.module [hunk 1]
```diff
@@ -17,42 +17,15 @@ define('TING_SEARCH_CAROUSEL_SEARCH_SIZE', 20);
...
     'title' => 'Show search carousel results',
     'page callback' => 'ting_search_carousel_result',
-    'page arguments' => array(3, 4),
+    'page arguments' => array(3, 4, 5),
     'access arguments' => array('access content'),
     'file' => 'ting_search_carousel.pages.inc',
     'type' => MENU_CALLBACK,
   );
 
-  $items['admin/config/ting/ting_search_carousel'] = array(
```
Comment (xendk) [inline modules/ting_search_carousel/ting_search_carousel.module]: The `configure[]` in the info file needs to go too.

**Comment by DDBdrift** [comment]
https://platform.dandigbib.org/issues/1744 | Resolved (tag version)

**Pull request closed without merging**